frontend/src/index.html

```diff
@@ -242,95 +242,119 @@ <h3 class="flex-grow break-words font-semibold" title="{{title}}">{{#helpers.hig
+                      />
+                    </svg>
+                  </button>
+                </div>
+                <div>
+                  <div class="volume-slider hidden items-center sm:mt-1.5 md:flex">
+                    <button id="volume-control">
+                      <svg xmlns="http://www.w3.org/2000/svg" height="24px" viewBox="0 0 24 24" width="24px" fill="#4b5563">
+                        <path d="M0 0h24v24H0V0z" fill="none" />
+                        <path
+                          d="M3 9v6h4l5 5V4L7 9H3zm7-.17v6.34L7.83 13H5v-2h2.83L10 8.83zM16.5 12c0-1.77-1.02-3.29-2.5-4.03v8.05c1.48-.73 2.5-2.25 2.5-4.02zM14 3.23v2.06c2.89.86 5 3.54 5 6.71s-2.11 5.85-5 6.71v2.06c4.01-.91 7-4.49 7-8.77 0-4.28-2.99-7.86-7-8.77z"
+                        />
+                      </svg>
+                      <svg xmlns="http://www.w3.org/2000/svg" height="24px" viewBox="0 0 24 24" width="24px" fill="#4b5563" class="hidden">
+                        <path d="M0 0h24v24H0V0z" fill="none" />
+                        <path
+                          d="M4.34 2.93L2.93 4.34 7.29 8.7 7 9H3v6h4l5 5v-6.59l4.18 4.18c-.65.49-1.38.88-2.18 1.11v2.06c1.34-.3 2.57-.92 3.61-1.75l2.05 2.05 1.41-1.41L4.34 2.93zM10 15.17L7.83 13H5v-2h2.83l.88-.88L10 11.41v3.76zM19 12c0 .82-.15 1.61-.41 2.34l1.53 1.53c.56-1.17.88-2.48.88-3.87 0-4.28-2.99-7.86-7-8.77v2.06c2.89.86 5 3.54 5 6.71zm-7-8l-1.88 1.88L12 7.76zm4.5 8c0-1.77-1.02-3.29-2.5-4.03v1.79l2.48 2.48c.01-.08.02-.16.02-.24z"
+                        />
+                      </svg>
+                    </button>
+                    <input type="range" id="volume" name="volume" min="0" max="10" value="0" class="volume-slider w-24 h-1 bg-gray-100 rounded-full focus:outline-none appearance-none" />
+                  </div>
+                </div>
               </div>
-            </div>
-            <div class="w-full md:mx-auto md:w-3/4">
-              <div class="seek-slider flex items-center">
-                <span class="teal-500 mr-2 text-xs">20:43</span>
-                <input type="range" id="progress" name="volume" min="0" max="100" value="0" class="track-slider relative w-full h-1.5 bg-gray-100 rounded-full focus:outline-none appearance-none" />
-                <span class="teal-500 ml-2 text-xs">00:00</span>
+              <div class="w-full md:mx-auto md:w-full">
+                <div class="relative w-full">
+                  <div class="flex-grow break-words font-semibold overflow-x-auto">
+                    <div class="absolute -left-1 top-0 w-10 h-6 bg-fade transform rotate-180 sm:hidden"></div>
+                    <div class="sm:whitespace-wrap whitespace-nowrap">
+                      <span class="mr-4"> </span>
+                      Q&A - The Love Between Krsna and Vrajavasis, How to Attain Perfection, etc.
+                      <span class="ml-8"> </span>
+                    </div>
+                    <div class="absolute -right-1 top-0 w-20 h-6 bg-fade sm:hidden"></div>
```

Comment:
I see you are using some helper elements, probably for fading the text. Can you give me a link of this solution in the internet? I would prefer if we avoid extra elements, so I want to research a bit on this.